main.py

```diff
@@ -115,6 +115,15 @@ def get_redirect_route(regex_route, handler, defaults=None):
 
     Warning: this method strips off parameters after the trailing slash. URLs
     with parameters should be formulated without the trailing slash.
+
+    Args:
+        regex_route: string:  A raw string representing a route.
```

Comment:
Please follow the format of existing docstrings. The typeinfo should be a valid Python type or a class name, and should be followed by a period and a single space. "string" is not a valid Python type -- it's either str/unicode. (Use `type()` to determine which.)

Reply:
Done. The type is of unicode.